src/main/java/stream/flarebot/flarebot/commands/general/TagsCommand.java

```diff
@@ -39,10 +39,15 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                     guild.getTags().remove(args[1]);
                     MessageUtils.sendSuccessMessage("You successfully removed the tag `" + args[1] + "`!",
                             channel, sender);
-                } else
+                } else {
                     MessageUtils.sendErrorMessage("This tag doesn't exist!", channel);
-            } else
+                }
+            } else if (args[0].equalsIgnoreCase("edit")) {
+                MessageUtils.sendErrorMessage("This seems to be invalid ¯\\_(ツ)_/¯\n"
```

Comment:
Nah it doesn't really matter, it wouldn't be quite weird as well to have normal chars then the full unicode Java code rather than just the Unicode itself. Java files are UTF-8 so it doesn't matter much. Plus Lang soon so it will be generally the same result.